test/saml/src/test/java/org/alfresco/identity/service/saml/test/TokenTest.java

```diff
@@ -0,0 +1,330 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * @author Jared Ottley
+ */
+@TestInstance(Lifecycle.PER_CLASS)
+ public class TokenTest
+{  
+    private Logger logger = LoggerFactory.getLogger(TokenTest.class);
+    private Properties appProps = null;
+    private ChromeDriverService service = null;
+
+    @BeforeAll
+    void setup()
+    {
+        try
+        {
+        String rootPath = Thread.currentThread().getContextClassLoader().getResource("").getPath();
+        String appConfigPath = rootPath + "application.properties";
+ 
+        appProps = new Properties();
+        appProps.load(new FileInputStream(appConfigPath));
+        }
+        catch (IOException exception)
+        {
+            logger.info("Unable to read properties file");
+        }
+    }
+
+    @AfterAll
+    void cleanup()
+    {
+        //Clean up remote chromedriver service
+        if (service != null)
+        {
+            service.stop();
+        }
+    }
+
+    @Test
+    void getTokenFromSAMLLogin() 
+    throws MalformedURLException, 
+           IOException, 
+           NoSuchAlgorithmException, 
+           InvalidKeySpecException, 
+           Exception
+    {   
+        //Create HTMLUnit WebDriver
+        WebDriver driver;
+
+        if(!isBrowserEnable())
+        {
+            driver = new HtmlUnitDriver(true);
+        }
+        else
+        {
+            ChromeDriverService service = new ChromeDriverService.Builder().usingAnyFreePort().build();
+            service.start();
+            driver = new RemoteWebDriver(service.getUrl(), DesiredCapabilities.chrome());
+        }
+        
+        //Initiate page
+        driver.get("https://" + getHostname() + "/auth/realms/" + getRealm() +"/protocol/openid-connect/auth?response_type=id_token%20token&client_id=alfresco&state=CIteJYtFrA22JnCikKHJ2QPrNuGHzyOphE1SsSNs&redirect_uri=http%3A%2F%2Flocalhost%2Fdummy_redirect&scope=openid%20profile%20email&nonce=CIteJYtFrA22JnCikKHJ2QPrNuGHzyOphE1SsSNs");
+       
+        //Click on SAML link on login page
+        WebElement element = driver.findElement(ByLinkText.linkText(TokenTestConstants.ELEMENT_SAML));
+        element.click();
+
+        //Select User, Enter password, and submit form on SAML page
+        Select select = new Select(driver.findElement(ByName.name(TokenTestConstants.ELEMENT_USERID)));
+        select.selectByVisibleText(getUser());
+        WebElement passwordfield = driver.findElement(ByName.name(TokenTestConstants.ELEMENT_PASWORD));
```

Comment:
should we modify the  `passwordfield` to corespond to the camel case standard?

Reply:
Thanks for spotting this, I've fixed it in 6159d3f.